installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

```diff
@@ -42,7 +42,7 @@ include::modules/ipi-install-modifying-install-config-for-dual-stack-network.ado
 include::modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
-== Additional resources
+=== Additional resources
```

Comment:
When used to avoid linking from module to module, I believe it should be `.Additional resources`